activerecord/lib/rails/generators/active_record/model/templates/abstract_base_class.rb.tt

```diff
@@ -0,0 +1,7 @@
+<% module_namespacing do -%>
+class <%= abstract_class_name %> < ApplicationRecord
+  self.abstract_class = true
+
+  connects_to database: { writing: :<%= database -%> }
```

Comment:
I was wondering if the `writing` key should use `ActiveRecord::Base.writing_role` instead since `ActiveRecord::Base.writing_role` is configurable.

Reply:
Good call, will update. I also need to fix the test failures and will add this when I do that. 😄